kernel/content/attribute_edit.php

```diff
@@ -221,7 +221,7 @@
     $postMaxSize = trim( ini_get( 'post_max_size' ) );
 
     // As of PHP 5.3.2 it is possible to configure unlimited post_max_size, further checking
-    if ($postMaxSize == 0)
+    if ($postMaxSize != 0)
     {
         $postMaxSizeBytes = $postMaxSize;
```

Comment:
This could help for the failing test:
`$postMaxSizeBytes = (int)$postMaxSize;`

Reply:
👍  , but I'll first var dump what the hell is configured here. maybe it's 0b or something, but have to know 😆

Reply:
It fails in line 239. So it is something with 'm' at the end :)